### ISSUE TITLE
Home for onyx droidmedia

### ISSUE THREAD

**Issue by rinigus** [kimmoli/sfos-onyx-issues#53]
While we have "canonical" repositories for several onyx SFOS port parts, we miss our specific droidmedia repo. Unfortunately, onyx requires some changes to the upstream and @kimmoli 's PR was not merged there. Right now, I made a separate branch for each release at https://github.com/rinigus/droidmedia/branches . OBS is building against the corresponding one.

@kimmoli , maybe we should use your repository as a canonical one to build against? Then I could submit all my changes for review and we have a process established.

**Comment by kimmoli** [comment]
I have missed this one totally. Is this still to-do?

**Comment by rinigus** [comment]
Yes, would be good to have some place where we could send PRs for gst-droid and droidmedia. They both have your specific onyx patches and right now I build from my repos, for hal packages. No rush, but probably some easy to find place is needed. Although, it kind of works so far
